src/core/ucc_constructor.c

```diff
@@ -53,6 +55,19 @@ ucc_status_t ucc_constructor(void)
             get_default_lib_path();
         }
     }
+
+    if (!ucc_global_config.component_path) {
+        //TODO
+        //ucc_error, "Failed to get ucc library path. set UCC_COMPONENT_PATH.\n");
+        return UCC_ERR_NO_MESSAGE;
+    }
+    if (UCC_ERR_NO_MESSAGE ==
```

Comment:
Why checking only for UCC_ERR_NO_MESSAGE?

Reply:
no reason, fixed